metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          "All rights reserved"
 description      "Installs/Configures repose"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "2.10.0"
+version          "2.10.1"
```

Comment:
Seems like a nit, but I would make this a minor release (`2.11.0`) versus a bug release (`2.10.1`).